deploy/03_deploy_lbp_contracts.js

```diff
@@ -1,14 +1,23 @@
+const { getBalancerContractAddress } = require("@balancer-labs/v2-deployments");
+
 const deployFunction = async ({ getNamedAccounts, deployments, ethers }) => {
   const { deploy } = deployments;
   const { root } = await getNamedAccounts();
   const safeInstance = await ethers.getContract("Safe");
-  const lbpFactoryInstance = await ethers.getContract(
-    "LiquidityBootstrappingPoolFactory"
+  const liquidityBootstrappingPoolFactoryTaskId =
+    "20210721-liquidity-bootstrapping-pool";
+  const contractName = "LiquidityBootstrappingPoolFactory";
+  const networkName = hre.network.name;
```

Comment:
funky stuff 💪🏼

Reply:
btw you can get the `network`object the same way you get the `ethers` and `deployments` objects in line 3

Reply:
Yeah just came accros the deployment package from Balancer and it worked great :D